app/serializers/move_serializer.rb

```diff
@@ -34,10 +34,11 @@ class MoveSerializer < ActiveModel::Serializer
     prison_transfer_reason
     court_hearings
     allocation
+    allocation.moves
```

Comment:
these will not appear in the GET all moves endpoint so there is no worry we'll return very large include responses